marklogic-data-hub-central/ui/e2e/Dockerfile

```diff
@@ -5,5 +5,6 @@ COPY ./cypress ./cypress
 COPY ./qa-project ./qa-project
 COPY ./cypress.json ./cypress.json
 COPY ./tsconfig.json ./tsconfig.json
+COPY ../src/services/user-preferences ./src/services/user-preferences
```

Comment:
The test will still fail to compile. Cypress is not in the same level as src
ui
--e2e
----cypress
--src

Reply:
maybe also change line 4 - 7 for the destination to be under e2e

Reply:
Resolved this. Please check now